core/block.ts

```diff
@@ -939,6 +939,25 @@ export class Block implements IASTNodeLocation, IDeletable {
     return this.disposed;
   }
 
+  /**
+   * @returns True if this block is a value block with a single editable field.
+   * @internal
+   */
+  isSimpleReporter(): boolean {
+    if (!this.outputConnection) return false;
+
+    let nFields = 0;
+    for (const input of this.inputList) {
+      if (input.connection) return false;
+      // TODO: This comment is innacurate. Not sure what we want the spec to be.
```

Comment:
I think you could just fix the comment. i assume by "text" it means "label" but it doesn't actually make sense to exclude those? like the full block rendering wouldn't make sense if there was a label. so i'd just update the comment.